src/components/utils/errors.js

```diff
@@ -0,0 +1,8 @@
+export const errors = {
```

Comment:
this is UI code specific to the sdk, it's not really an utility. Utilities then to be general purpose tools, not business specific.